spark/src/test/scala/org/apache/comet/CometCastSuite.scala

```diff
@@ -104,6 +104,14 @@ class CometCastSuite extends CometTestBase with AdaptiveSparkPlanHelper {
     castTest(generateBools(), DataTypes.IntegerType)
   }
 
+  test("cast double to string") {
+    castTest(generateDoubles(), DataTypes.StringType)
+  }
+
+  test("cast float to string") {
```

Comment:
```suggestion
  test("cast FloatType to StringType") {
```

Reply:
I saw that there were already defined functions for both types so I moved them there